### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by tonylodge** [code-ready/crc#1320]
### General information

  * OS: macOS
  * Hypervisor: KVM / Hyper-V / hyperkit
  * Did you run `crc setup` before starting it (Yes/No)? yes
  * Running CRC on: Laptop 

## CRC version
```bash
# crc version: 1.11.0+883ca49
```
  
## CRC status
```bash
# Put `crc status --log-level debug` output here
```

## CRC config
```bash
# fails on crc start 
```

## Host Operating System
```bash
ProductName:	Mac OS X
ProductVersion:	10.15.5
BuildVersion:	19F101
```

### Steps to reproduce

  crc start
INFO Checking if oc binary is cached              
INFO Checking if podman remote binary is cached   
INFO Checking if goodhosts binary is cached       
INFO Checking if running as non-root              
INFO Checking if HyperKit is installed            
INFO Checking if crc-driver-hyperkit is installed 
INFO Checking file permissions for /etc/resolver/testing 
? Image pull secret [? for help] ************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************INFO Loading bundle: crc_hyperkit_4.4.5.crcbundle ... *****************************************************************************************************************************************************INFO Checking size of the disk image /Users/tonylodge@ca.ibm.com/.crc/cache/crc_hyperkit_4.4.5/crc.qcow2 ... **********************************************************************************************INFO Creating CodeReady Containers VM for OpenShift 4.4.5... **********************************************************************************************************************************************ERRO Error creating host: Error creating the VM: Error creating machine: Error in driver during machine creation: getting MAC address from UUID: vmnet: error from vmnet.framework: -1

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by gbraad** [comment]
Did you restart your machine?

As there might be a stray process still running that claims the same UUID.

Duplicate of #882, #1310
